### DISCUSSION TITLE
Thoughts about lexical match pipeline

### DISCUSSION THREAD

**Discussion by matentzn** [monarch-initiative/mondo-ingest#40] [Ideas]
This is the accompanying discussion on https://github.com/monarch-initiative/mondo-ingest/issues/38. I started a discussion to make it easier to comment on individual ideas.

**Comment by matentzn** [comment]
We should restrict matching to looking across specif prefix spaces. In the mondo.sssom.config.yaml file linked in the PR, you see the FROM, TO lists below. We only care about these: MONDO - NCIT, MONDO - OMIM, MONDO - OMIMPS etc. Is this something we can set in the rules file?

**Reply by hrshdhgd** [reply]
[These](https://incatools.github.io/ontology-access-kit/datamodels/mapping-rules/Precondition.html#slots) seem to be that only rules that are available. Am I right, @cmungall ?

So the options available are:

```
mapping_source_one_of
object_match_field_one_of
object_source_one_of
subject_match_field_one_of
subject_source_one_of
transformations_included_in
```

**Reply by matentzn** [reply]
This looks great, lets add one of these rules files to the PR here.
